subprojects/core/src/integTest/groovy/org/gradle/api/tasks/CopyPermissionsIntegrationTest.groovy

```diff
@@ -222,4 +223,35 @@ class CopyPermissionsIntegrationTest extends AbstractIntegrationSpec {
         cleanup:
         file('src/unauthorized').mode = 0777
     }
+
+    @Requires(TestPrecondition.FILE_PERMISSIONS)
+    @Issue('https://github.com/gradle/gradle/issues/9576')
+    def "unreadable #type not produced by task is ignored"() {
```

Comment:
Do we have a test for what should happen with unreadable content produced by the task?

Reply:
We can do that in a follow-up, as suggested by @wolfs: https://github.com/gradle/gradle/pull/9746#issuecomment-505865852